unicodetools/src/main/java/org/unicode/props/ShimUnicodePropertyFactory.java

```diff
@@ -0,0 +1,392 @@
+package org.unicode.props;
+
+import com.ibm.icu.dev.util.UnicodeMap;
+import com.ibm.icu.impl.locale.XCldrStub.Splitter;
+import com.ibm.icu.text.NumberFormat;
+import com.ibm.icu.text.UTF16;
+import com.ibm.icu.text.UnicodeSet;
+import com.ibm.icu.util.ULocale;
+import java.lang.reflect.Method;
+import java.util.function.BiFunction;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import org.unicode.cldr.util.Rational;
+import org.unicode.props.UcdPropertyValues.Idn_Status_Values;
+import org.unicode.props.UcdPropertyValues.Script_Values;
+import org.unicode.props.UnicodeProperty.BaseProperty;
+import org.unicode.text.UCD.Normalizer;
+import org.unicode.text.UCD.Normalizer.NormalizationFormat;
+import org.unicode.text.utility.Utility;
+
+public class ShimUnicodePropertyFactory extends UnicodeProperty.Factory {
```

Comment:
This could use a comment explaining what it is for (as far as I can tell it makes an IUP behave like TUP for testing/diffing purposes, but we shouldn’t actually be used beyond that, right?).

Reply:
done